# Q1

In [1]:
c1 = c(5.53,26.11,11.08,12.66,0.87,6.40,54.31,37.94,7.13,3.53,14.23,1.24)
c2 = c(4.84,10.93,0.65,32.52,3.56,11.68,35.97,47.07,17.69,40.7)
c3 = c(15.79,11.18,27.54,28.11,21.70,1.25)

In [2]:
Y1 = sum(c1)

In [3]:
Y1

[1] 181.03

In [4]:
Y2 = sum(c2)

In [5]:
Y2

[1] 205.61

In [6]:
Y3 = sum(c3)

In [7]:
Y3

[1] 105.57

In [9]:
paste(Y1,Y2,Y3)

[1] "181.03 205.61 105.57"

In [16]:
Yc_bar = sum(Y1,Y2,Y3)/sum(length(c1),length(c2),length(c3))
# Overall cluster size is 30

In [17]:
Yc_bar

[1] 17.57893

Thus the average yield of apple per tree is 17.5789 kgs.